Source/itemdat.h

```diff
@@ -282,6 +282,8 @@ enum item_cursor_graphic : uint8_t {
 	ICURS_BLITZEN                     = 219,
 	ICURS_DEMON_PLATE_ARMOR           = 225,
 	ICURS_BOVINE                      = 226,
+
+	ICURS_DEFAULT  = 1 << (sizeof(item_cursor_graphic) - 1),

```

Comment:
https://github.com/diasurgical/devilutionX/blob/039a6c91bc85e7106da94c4710fc3a305de3412f/Source/monster.h#L70
guess we should be fine with doing the same thing for consistency